src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.css

```diff
@@ -1,35 +1,70 @@
 .modalSavePressetesWindow {
-    position: fixed;
-    top: 50%;
-    margin-top: -100px;
-	margin-left: -200px; 
-	left: 50%;      
-    background-color: #2ecc71;
-    width: 400px;
-    height: 200px;
-    box-sizing: border-box;
-    z-index: 101;
+  color: var(--mainText-ordinary-white);
+  border-radius: 8px;
+  position: fixed;
+  top: 50%;
+  margin-top: -100px;
+  margin-left: -200px;
+  left: 50%;
+  background-color: var(--headerBackground-dark-super);
+  width: 400px;
+  height: 200px;
+  box-sizing: border-box;
+  letter-spacing: 1.5px;
+  z-index: 101;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
 }
 
 .headerModalWindow {
-	text-align: right
+  text-align: right
 }
 
-.textArea {
-padding-left: 20px;
+.headerModalWindow .icon {
+ margin: 10px;
+}
+
+.textArea strong {
```

Comment:
Don't chage styles in standart styled tags. Use span, for example.

Reply:
Done)